lib/muchos/config.py

```diff
@@ -65,6 +65,11 @@ def verify_config(self, action):
                     if not self.has_service(service):
                         exit("ERROR - Missing '{0}' service from [nodes] section of muchos.props".format(service))
 
+            # validate if we are using Java 11 and fail if we are using Accumulo 1.x
+            # See https://github.com/apache/accumulo/issues/958 for details
+            if self.java_product_version() == 11 and self.version('accumulo').split('.')[0] == '1':
+                exit("ERROR - Java 11 is not supported with Accumulo version '{0}'".format(self.version('accumulo')))
```

Comment:
My memory is a bit fuzzy on this.  I thought some work was done to make 1.9 work with Java 11.  Maybe that will be for 1.9.4.  This is fine for now, may want to revisit this in the future.

Reply:
Thanks Keith. We tested with 1.9.3 and Java 11, what failed was the build of the native libraries. Seemingly there is something in that Makefile which is tripping up, because it kept complaining about "no Java SDK at JAVA_HOME". That must be an Accumulo issue, I can open an issue on that project to track it separately.

Reply:
An Accumulo issue for that would be nice.

Reply:
@keith-turner I checked and looks like you have already (fixed)[https://github.com/apache/accumulo/commit/9591cc2b4b00897894e498d7a4cf49d63edbe7fc] this in 1.9 master. I presume that fix will make it to 1.9.4 so on the Muchos side, I will revise my check to specifically check for 1.9.3 or less and then block if Java 11 (or higher) is in use.

Again, please hold off on merging this till I make those validations real.

Reply:
I didn't even remember that, thanks for the link.  One of my goals is to make 1.9.4 work with Java 11 in order to provide a more gradual upgrade path.  People can upgrade to Java 11 while still using Accumulo 1.9, not having to upgrade to Java 11, Hadoop 3, and Accumulo 2 all at once.  I did a round of testing with Java 11 in the past and just fixed whatever issues I found.  Hopefully I can do that again before 1.9.4 is released.

Reply:
Hi @keith-turner I made the check more robust now. Please have a look.

Reply:
The changes look great.

Reply:
Thank you, Keith, we can proceed to merge the changes.